src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -291,6 +297,12 @@ module.exports = function<T, P, I, TI, C>(
         commitTextUpdate(textInstance, oldText, newText);
         return;
       }
+      case Portal: {
+        const children = finishedWork.output;
```

Comment:
We can just do `const children = finishedWork;` for now since the host config will just traverse the tree deeply right now.